packages/botonic-react/src/webchat/webchat-chat-area/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useContext, useEffect, useState } from 'react'
+
+import { WebchatContext } from '../../contexts'
+import { BotonicContainerId } from '../constants'
+import { useWebchatDimensions } from '../hooks'
+import { WebchatMessageList } from '../message-list'
+import { WebchatReplies } from '../replies'
+import { StyledWebchatChatArea } from './styles'
+
+export const WebchatChatArea = () => {
```

Comment:
for a future refactor rename this component to ChatArea and WebchatInputPanel to InputPanel

Reply:
next PR